src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -125,31 +125,92 @@ public static void ClientMessageInspector_Verify_Invoke()
 
     [Fact]
     [OuterLoop]
-    public static void ClientBaseOfT_Call()
+    public static void ClientBaseOfT_Sync_RoundTrip_Check_CommunicationState()
     {
-        // This test verifies ClientBase<T> can be used to create a proxy and invoke an operation
-        StringBuilder errorBuilder = new StringBuilder();
+        CustomBinding customBinding = new CustomBinding();
+        customBinding.Elements.Add(new TextMessageEncodingBindingElement());
+        customBinding.Elements.Add(new HttpTransportBindingElement());
+
+        MyClientBase client = new MyClientBase(customBinding, new EndpointAddress(Endpoints.HttpSoap12_Address));
+        Assert.Equal(CommunicationState.Created, client.State);
+
+        IWcfServiceGenerated serviceProxy = client.ChannelFactory.CreateChannel();
+        Assert.Equal(CommunicationState.Opened, client.State);
+
         try
         {
-            CustomBinding customBinding = new CustomBinding();
-            customBinding.Elements.Add(new TextMessageEncodingBindingElement());
-            customBinding.Elements.Add(new HttpTransportBindingElement());
+            string result = serviceProxy.Echo("Hello");
+            Assert.Equal(CommunicationState.Opened, client.State);
 
-            MyClientBase client = new MyClientBase(customBinding, new EndpointAddress(BaseAddress.HttpBaseAddress));
-            IWcfServiceGenerated serviceProxy = client.ChannelFactory.CreateChannel();
+            // Not in profile for CoreCLR / .NET Native
+            // client.Close();
```

Comment:
I was on the fence about this one. I *know* that I'll forget that I thought about this already the next time I see this code :smile: 

I don't necessarily care to Close(), I just want an indication that it's been thought about...